src/index.ts

```diff
@@ -2084,13 +2084,23 @@ export class GossipSub extends EventEmitter<GossipsubEvents> implements Initiali
     }
   }
 
+  /**
+   * Emits gossip - Send IHAVE messages to a random set of gossip peers. This is applied to mesh
```

Comment:
I'm pretty sure we _don't_ gossip to mesh peers, but rather only to non-mesh peers (that meet the other criteria)

Reply:
the comment does not go with the code, will fix it 👍